Assets/Scripts/BowlingBall.cs

```diff
@@ -1,151 +0,0 @@
-﻿using System;
-using System.Collections;
-using System.Collections.Generic;
-using System.Security.AccessControl;
-using UnityEngine;
-using UnityEngine.EventSystems;
-using UnityEngine.Experimental.UIElements;
-using UnityEngine.XR.WSA;
-
-public class BowlingBall : MonoBehaviour
-{
-
-	// Use this for initialization
-	public Vector3 InitialVelocity = Vector3.forward;
-	public Vector3 InitialPosition;
-	public float TranslationCoefficient;
-
-	private bool isMoving;
-	private Rigidbody ballRigidbody;
-	private AudioSource audioSource;
-	private string translationDirection;
-	private const float LANE_BOUNDS = 0.45f;
-	private Vector3 GUTTERBALL_BOUNDS = new Vector3(1, -1, 27);
-	private bool readyToLaunch;
-	private bool hasLaunched;
-	private bool ballEnteredBox;
-
-
-
-
-	void Start()
-	{
-		ballRigidbody = GetComponent<Rigidbody>();
-		audioSource = GetComponent<AudioSource>();
-		ballRigidbody.useGravity = false;
-	}
-
-	private void Update()
-	{
-
-		if (translationDirection == "right")
```

Comment:
Would be less error prone to replace these string literals with constants used both here, and where the ```TranslateDirection()``` method is called.